packages/create-quip-app/bin/create-quip-app.py

```diff
@@ -95,7 +95,7 @@ def replace_placeholders(file_path):
             f.write(content.replace("$APP_DIR_NAME", app_dir_name))
             f.truncate()
 
-    for file_path in ["app/manifest.json", "package.json"]:
+    for file_path in ["manifest.json", "package.json"]:
```

Comment:
May want to support both just so if/when people blindly npm i this in existing projects and it's tagged `@latest` they don't encounter an error

Reply:
Updated to use quip-cli init in the most recent push